### ISSUE TITLE
/>

### ISSUE THREAD

**Issue by tomByrer** [tonsky/FiraCode#115]
`/>` looks very bad, hard to read & feels forced.  Since I see this mostly in HTML, I find the slash to be noisy anyhow.   I'd prefer a version with the `/` is smaller but with a space before `>`.  Let me know if you want a mockup.

**Comment by MadcapJake** [comment]
I'd say if anything, raise the slash a little more so it's more obvious it's too characters combined.  But honestly, I don't really see a problem with it.

Pertinent to add that I'm pretty sure the ending slash isn't even required in HTML5.

**Comment by tomByrer** [comment]
>  I'm pretty sure the ending slash isn't even required in HTML5.

True, but some auto-complete & rendering engines do, so I want to catch those.  Plus it is still standard in XML, thus SVG which I use often.

**Comment by tonsky** [comment]
@tomByrer do you even like something about this font at all? Looks like you’re just asking to turn everything off. What will remain?

**Comment by tomByrer** [comment]
I'm not sure where you got that idea?  Sorry if I made you feel bad.

**Comment by easyaspi314** [comment]
@tonsky 
About this ligature, how about a scalene triangle? Like the one for `|>` but angled?

**Comment by tonsky** [comment]
It would be either too small (it has to take two horizontal places), or
triangle would be too big and out of proportions with opening bracket

**Comment by easyaspi314** [comment]
Well what about just aligning the slash to the caret? Instead of that weird
overlap ?

**Comment by tonsky** [comment]
How do you mean? A screenshot maybe?

**Comment by easyaspi314** [comment]
This is roughly what I mean. I did this from my phone, so it is a little sloppy.

![tmp_31039-closing slash-595597593](https://cloud.githubusercontent.com/assets/6258309/14491055/91f3ebfc-0144-11e6-817f-c5b2bedf21e2.png)

**Comment by tonsky** [comment]
I got the idea, thanks. I think I like it the way it is now, sorry :)

**Comment by easyaspi314** [comment]
OK.
